arm-network/2016-12-01/swagger/virtualNetworkGateway.json

```diff
@@ -1521,6 +1521,17 @@
           "type": "boolean",
           "description": "EnableBgp flag"
         },
+        "UsePolicyBasedTrafficSelectors": {
```

Comment:
@amarzavery I'm actually putting the changes in 2016-12-01 so that it's easier to see which parts are being changed through diff. I have discussed with @deepakrajendranmsft already about how to move to new api-version - I will move all the changes to a new api version once everything is clear.